lib/LessStringifier.js

```diff
@@ -9,7 +9,13 @@ module.exports = class LessStringifier extends Stringifier {
 
     let name = `${node.raws.identifier || '@'}${node.name}`;
     let params = node.params ? this.rawValue(node, 'params') : '';
-    const important = node.raws.important || '';
+    let important = '';
+
+    if (node.important && node.raws.important) {
```

Comment:
maybe I'm not reading the code above correctly, but in what scenario will `node.raws.important` be populated when `node.important` is falsy?